radixdlt/babylon-wallet-android
# Implement radix banner in wallet screen

## Description
When the user creates a new wallet we present them with the banner that links to https://wallet.radixdlt.com/. The user can only dismiss the banner by clicking on the X button. Users that don't create a new wallet but restore from backup, are not presented with this banner. The banner will be the last element of the list in wallet screen.


## How to test
1. Create a new wallet
2. The banner should be displayed, unless you dismiss it by clicking X

1. Restore from backup
2. You should not see the banner.

## Screenshot
<img src="https://github.com/radixdlt/babylon-wallet-android/assets/125959264/2a2e0f86-c227-448d-92d9-244c788c0bcd" width="300">

## PR submission checklist
